### DISCUSSION TITLE
How may I trigger completion in command mode?

### DISCUSSION THREAD

**Discussion by jrhorn424** [JetBrains/ideavim#305] [Q&A] [answered]
Let's say I'm in normal mode, and open the command line (is that what it's called?) with `:`. I have NERDTree, so I want to see what actions are available. Typically, I'd use `<tab>` in `nocompatible` `vim` to trigger a wildmenu. However, in IdeaVim this jumps the cursor back to the open buffer. I've also tried `<C-e>` as I've read that's the old `vi` completion key.

Is there a way to accomplish something similar in IdeaVim?

**Comment by leeturner** [comment, marked as answer]
Hi, I don't think that is supported at the moment.  There is this issue but it is quite old now - https://youtrack.jetbrains.com/issue/VIM-258

**Reply by jrhorn424** [reply]
🙏 Thanks! I clicked through to vote on the issue. I'd like to encourage others to do the same. ✅

**Reply by woodgear** [reply]
i know you feel.bro.
